test/forces.jl

```diff
@@ -1,10 +1,12 @@
 using DFTK
 using Test
+using Unitful
+using UnitfulAtomic
 include("testcases.jl")
 
 @testset "Forces on semiconductor (using total energy)" begin
     function energy(pos)
-        Ecut = 5
+        Ecut = 5 * u"hartree"
```

Comment:
Hmm. I think at this place this is not needed because this test is really testing the forces and not the Ecut / grid setup.